src/nvim/terminal.c

```diff
@@ -618,6 +621,17 @@ static int term_movecursor(VTermPos new, VTermPos old, int visible,
   return 1;
 }
 
+// Set the terminal title. Takes ownership of the title string.
```

Comment:
There is no ownership takeover for `title` when this function is called. The title string is copied.